apollo-common/src/main/java/com/ctrip/framework/apollo/common/utils/WebUtils.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2022 Apollo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.ctrip.framework.apollo.common.utils;
+
+import com.google.common.base.Splitter;
+import com.google.common.base.Strings;
+import javax.servlet.http.HttpServletRequest;
+
+/**
+ * @author kl (http://kailing.pub)
+ * @since 2022/8/12
+ */
+public class WebUtils {
+
+  private WebUtils(){}
+  private static final Splitter X_FORWARDED_FOR_SPLITTER = Splitter.on(",").omitEmptyStrings()
+      .trimResults();
```

Comment:
I think it's better to move static final variables before the constructor.

```suggestion
  private static final Splitter X_FORWARDED_FOR_SPLITTER = Splitter.on(",").omitEmptyStrings()
      .trimResults();

  private WebUtils(){}
```